acceptance/dockerImage/build/runCucumber.sh

```diff
@@ -17,6 +17,8 @@ CUCUMBER_CMD="cd /capybara; cucumber $*"
 eval ${CUCUMBER_CMD}
 EXIT=$?
 
-java -jar /usr/share/reporter/reporter.jar ./output ./output/report.json
-chown -R $CALLER_UID:$CALLER_GID ./output
-exit $EXIT
+if [ -d ./output ] && [ -f ./output/report.json ]; then
```

Comment:
Up at like 6, you could change to something like "if CAPYBARA_DRIVER is selenium or selenium-chrome, then start the xvfb".  This should work equally well with UI and API acceptance tests, and it will save a little time on the API acceptance tests by skipping the unnecessary step of starting xvfb